TeamForward/client/src/components/ProfilePage/ProfileImg.js

```diff
@@ -1,14 +1,14 @@
 import { useReactiveVar } from "@apollo/client";
-import { userState } from "../../GlobalState";
+import ImageIcon from "../ImageIcon";
 
 const ProfileImg = ({profileData, setProfileData}) => {
-  const user = useReactiveVar(userState);
+
   return (
     <div className="w-48 h-48 overflow-hidden bg-indigo-100 mx-auto rounded-full shadow-2xl absolute inset-x-0 top-0 -mt-24 flex items-center justify-center  text-indigo-500">
-      {profileData.cloudinaryProfileImgUrl ? (
-        <img
-          className="object-cover w-48 h-48 rounded-full"
-          src={profileData.cloudinaryProfileImgUrl}
+      {profileData.s3ProfilePhotoKey ? (
```

Comment:
here you can break this up into smaller components that are just internal to this file